src/docs/patterns/appbar.mdx

```diff
@@ -35,3 +37,37 @@ In some situations, you may want to present more information in an app bar, such
 <DemoCard repository={'search-bar'} angular react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'} ionic reactNative />
 
 A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white Search Bar slides in from the right to cover the default toolbar. As the user enters text into the search field, the data is filtered in real-time.
+
+<Divider />
+
+# Design Specifications
+
+For additional instructions, refer to [Material Design's design specs on "App bars: top"](https://material.io/components/app-bars-top#specs).
+
+## Mobile
+
+<ImageGrid images={[Specs1Image]} regularWidth />
+
+## Desktop and Tablet
+
+If certain dimensions are not specified, refer to the dimensions suggested in the mobile section above.
+
+<ImageGrid images={[Specs2Image]} regularWidth />
```

Comment:
For the appbar left icon, google seems to recommend 16px left and 32px right padding (which is what we've been implementing in our components):
![image](https://user-images.githubusercontent.com/29152776/88387272-22b4be80-cd80-11ea-9a2f-1b78d0432575.png)

However, their examples seem to be a mix of 16/32 and 24/24. But I didn't see any 24/32.

My preference would be to stick with 16/32 across the board to maintain the 72px keyline throughout the app. Angular Material and Material UI also both use 16px on the left side out-of-the-box, so if we're recommending 24, then every time someone uses an Appbar they're going to have to override the default styles.

This also begs the question...if we're following Material specs, do we even need to have a spec image here, or do we just refer them to the Material documentation?

Reply:
I must have it in my mind that docit is using 24/32 and thus I recommended 24 as left padding...

I can remove that 24/32, and claim docit is getting some "special treatment" :P

Reply:
Doc-It got special treatment - I can't remember what the exact issue was, but it was something to do with the negative margins of the GridList on a smaller width screen...I think the options were to refactor the grid list spacing, or just bump up the toolbar padding to 24 and we went with the easier approach.